mappings/net/minecraft/block/entity/BannerPatterns.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_7446 net/minecraft/block/entity/BannerPatterns
+	METHOD method_43721 initAndGetDefault (Lnet/minecraft/class_2378;)Lnet/minecraft/class_2582;
```

Comment:
```suggestion
	METHOD method_43721 registerAndGetDefault (Lnet/minecraft/class_2378;)Lnet/minecraft/class_2582;
```
See `MaterialRule` `MaterialCondition` `DensityFunctionTypes` `PaintingVariants`

Reply:
The `initAndGetDefault` method name is used for dimension types, structure sets, world presets, and flat level generator presets. While the inconsistency should be fixed, I think it is out of scope for this pull request.

Reply:
Hmm, turns out the `Registry` ones use `registerAndGetDefault` and `BuiltinRegistries` ones use `initAndGetDefault`

Reply:
Kinda makes sense given DRM registries are dynamic (although I believe `BuiltingRegistries` is not the ones they register to)